### PR TITLE
add new field, taking the existence of ropensci-archive into account

### LINKED ISSUE #43: New field request
I'd need a field for the website that'd be "type" for instance. 

* active if it needs to be listed in the active tab (probably all ropensci packages except if their status is different? but why have it in ropensci if it is not active?)
* experimental for ropenscilabs packages, also ropensci-org?
* archived for packages that are in ropensci-archive.

### PR DESCRIPTION
**Pull request by maelle**
Fix #43 (and few tweaks related to ropensci-archive)

### CODE REVIEW

File: R/registry.R [hunk 1]
```diff
@@ -57,20 +57,24 @@ get_status <- function(entry) {
     status <- guess_status(entry)
   }
   status <- gsub("http(s)?\\:\\/\\/www\\.repostatus\\.org\\/\\#",
-    "https://www.repostatus.org#", status)
+                 "https://www.repostatus.org#", status)
   return(status)
 }
 
 guess_status <- function(entry) {
```
Comment (maelle) [inline R/registry.R:64]: adding type was an opportunity to clean up `guess_status()` (early returns :-) )

File: R/registry.R [hunk 2]
```diff
@@ -202,6 +206,22 @@ create_registry <- function(cm, outpat, time = Sys.time()) {
   website_info$on_bioc <- purrr::map(website_info$name,
                                      get_bioc, bioc_names)
 
+  get_type <- function(status) {
```
Comment (maelle) [inline R/registry.R:209]: these "types" are the three tabs we have on https://roweb3-hugo.netlify.app/packages/all

**Comment by sckott** [comment]
does this failure indicate a problem? or not really? https://github.com/ropensci-org/makeregistry/pull/48/checks?check_run_id=1142593275#step:9:102

**Comment by maelle** [comment]
no because there's the same failure on master, but I really need to have a look. I'll download the artefacts and will then update the thread.

**Comment by maelle** [comment]
Fine to ignore, here's the difference between the snapshots.

image

I'll skip/tweak the test on master.

Any other comment on this PR?

**Comment by sckott** [comment]
okay. no other comment

**Review by sckott — APPROVED** [review]

**Pull request merged**